tests/flows/test_index.py

```diff
@@ -321,6 +324,123 @@ async def test_run_partial_updates_of_concepts_for_document_passages(
     assert test_counts_serialised == json.loads(result["Body"].read().decode("utf-8"))
 
 
+@pytest.mark.asyncio
+@pytest.mark.vespa
+@pytest.mark.slow
+async def test_run_partial_updates_of_concepts_for_document_passages_with_paging(
```

Comment:
Question: what's paging referring to here?

Reply:
It was the mixed term we were using about improving reading from Vespa.